web-bundle/src/main/java/com/graphhopper/replica/GtfsLinkMapper.java

```diff
@@ -26,6 +26,7 @@ public class GtfsLinkMapper {
     private final GraphHopper graphHopper;
     private final String CSV_COLUMN_HEADERS = "route_id,feed_id,stop_id,next_stop_id," +
             "stop_lat,stop_lon,stop_lat_next,stop_lon_next,street_edges,transit_edge";
+    private final String GTFS_LINK_MAPPING_DIR = "transit_data/gtfs_link_mappings/";
```

Comment:
I originally tried this, because you're right, it's used in several places, but then I balked when I realized we'd need to add a new maven dependency in the package `RouterService` lives in to access it. Maybe that's still worth it? It just seemed like a heavier change than I'd hoped (although adding a dep like this shouldn't _really_ be a big issue in many circumstances)